src/controller/watchlists.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const jwt = require('express-jwt');
+const { User } = require('../models');
+
+const watchlists = express.Router();
+const { ACCESS_TOKEN_SECRET: secret } = process.env;
+
+watchlists.post('/backup', jwt({ secret }), async (req, res) => {
+  try {
+    const { id } = req.user;
+    const user = await User.findById(id);
+    if (!Array.isArray(req.body)) return res.sendStatus(400);
```

Comment:
error handling should get more attention it can be confusing if you just hide it in the code with one line. people who run through the code won't notice this. But this is just my opinion if the team prefers these one liner if statements then go with them. But the simple `400` answer definitely not enough, in case of client error we always need to tell enough information to fix the problem